### PR TITLE
Annotation scrum 2016

### PR DESCRIPTION
**Pull request by Acribbs**
We have agreed that the annotation pipeline should be merged after we have decided to make a new GTF pipeline. I have done some testing and it seems to pass when I run each function.

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Many thanks to all that contributed!

I would appreciate a second opinion here before merging @CGATOxford/contributors

**Review by AndreasHeger — APPROVED** [review]
Looks all good, I suggest to merge.

**Comment by sebastian-luna-valero** [comment]
Thanks @AndreasHeger

**Pull request merged**
